src/lwip/silabs/lwipopts-wf200.h

```diff
@@ -70,19 +70,19 @@
 
 #define LWIP_SOCKET 0
 
-#define TCPIP_THREAD_PRIO osPriorityAboveNormal
-
-#ifdef DIC_ENABLE
+#ifdef MATTER_AWS_ENABLE
 #define LWIP_DNS 1
 #define DNS_RAND_TXID() ((u32_t) rand())
 #define MEM_SIZE 5632
 #define MEMP_NUM_UDP_PCB (6)
 #define TCP_MSS (4 * 1152)
+#define TCPIP_THREAD_PRIO osPriorityAboveHigh
```

Comment:
TODO: merge this priority as one for both the cases, does not make a lot of sense to have this different for AWS and non AWS feature set.